source/api_cc/src/DeepPotJAX.cc

```
@@ -0,0 +1,780 @@
+#define PADDING_FACTOR 1.05
+
+inline void check_status(TF_Status* status) {
+  if (TF_GetCode(status) != TF_OK) {
+    throw deepmd::deepmd_exception("TensorFlow C API Error: " +
+                                   std::string(TF_Message(status)));
+  }
+}
+
+inline void find_function(TF_Function*& found_func,
+                          const std::vector<TF_Function*>& funcs,
+                          const std::string func_name) {
```

cppcheck

[performance] Function parameter 'func_name' should be passed by const reference.

(passedByValue)